config/routes/candidate.rb

```diff
@@ -15,7 +15,8 @@
   end
 
   resources :account_recovery_requests, only: %i[new create], path: 'account-recovery-requests'
-  resources :invites, only: %i[index], path: 'application-sharing'
+  resources :invites, only: %i[index show], path: 'application-sharing'
```

Comment:
```suggestion
  resources :invites, only: %i[index show update], path: 'application-sharing'
```

I would try to use the `:invites` resource rather than doing it manually.